scripts/common.py

```diff
@@ -128,6 +132,11 @@
 FIREBASE_EMULATOR_CACHE_DIR = (
     os.path.join(CURR_DIR, os.pardir, 'firebase_emulator_cache'))
 
+sys.path.insert(0, PY_GITHUB_PATH)
+MYPY = False
+if MYPY:  # pragma: no cover
+    import github # isort:skip  pylint: disable=wrong-import-position
```

Comment:
Why is this necessary? Please add a comment explaining why

Reply:
Oops, We don't need this comment to skip pylint check. I have removed this and thanks for pointing it out.